client/src/components/Feed/Post.js

```diff
@@ -74,7 +74,7 @@ const URLS = {
 const filters = Object.values(filterOptions);
 
 const Highlight = ({ text = "", highlight = "" }) => {
-  if (!highlight || !highlight.trim()) {
+  if (!highlight || !highlight.trim() || !(typeof text === "string")) {
```

Comment:
text property will be `ReactComponent` when including hyperlink, which cannot do `split()`, this might not be the ideal solution, currently disable the highlighting for those text including hyperlinks, let me know how could we maintain the `text` to be string when involving hyperlinks, or other refactor solutions

Reply:
@ me , this won't work, all post title and body are ReactComponent right now, so won't see any highlight, will try something else